src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -91,55 +93,76 @@ const RadioFrequencyLink = styled.a`
   }
 `;
 
-const CanonicalRadioSchedule = ({ endpoint }) => {
-  const [schedule, setRadioSchedule] = useState();
+const CanonicalRadioSchedule = ({ initialData, endpoint }) => {
   const {
     service,
     script,
     dir,
     timezone,
     locale,
-    radioSchedule,
+    radioSchedule: radioScheduleConfig,
     translations,
   } = useContext(ServiceContext);
 
   const { timeOnServer } = useContext(RequestContext);
-  const header = pathOr(null, ['header'], radioSchedule);
+
+  const [radioSchedule, setRadioSchedule] = useState(initialData);
+
+  const header = pathOr(null, ['header'], radioScheduleConfig);
   const frequenciesPageUrl = pathOr(
     null,
```

Comment:
There is a simpler `path` function for when we don't need to specify a fallback value: https://ramdajs.com/docs/#path

`path` falls back to undefined - I don't think `null` vs `undefined` makes a difference in most instances here

Reply:
I've just been looking at this too - I'll update the usage here.

Reply:
Actually as all of these examples are just one level deep into the object, I should be fine to default `radioScheduleConfig` to an empty object, and then destructure these values out.

Reply:
Done! Have updated usages across this PR.